server/server/Models/restaurant.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+// schema
+const restaurantSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+
+  name: {
+    type: String,
+    required: true,
+  },
+
+  location: {
+    type: [String],
+    required: true,
+  },
+
+  phoneNumber: {
+    type: String,
+    required: true,
+  },
+
+  restaurantPicture: {
```

Comment:
this field will be string which represents the path to image file inside the backend

Reply:
makes sense, i'll edit it